addon/mixins/model-records/index-controller-mixin.js

```diff
@@ -15,10 +16,18 @@ export default Mixin.create({
   page: 1,
   size: 30,
 
-  'filter[keyword]': '',
+  'filter[keyword]': computed(() => ''),
```

Comment:
it ensures that the prop does not leak state between instances.  You can do this or set the property on init.  Here is a good article that talks about leakage:
https://dockyard.com/blog/2015/09/18/ember-best-practices-avoid-leaking-state-into-factories